boilerplate/src/app/index.tsx

```diff
@@ -1,84 +1,5 @@
-import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
+import { WelcomeScreen } from '@/screens/WelcomeScreen'
 
-import { Screen } from "@/components/Screen"
-import { Text } from "@/components/Text"
-import { isRTL } from "@/i18n"
-import { ThemedStyle } from "@/theme"
-import { useAppTheme } from "@/utils/useAppTheme"
-import { useSafeAreaInsetsStyle } from "@/utils/useSafeAreaInsetsStyle"
-
-const welcomeLogo = require("@assets/images/logo.png")
-const welcomeFace = require("@assets/images/welcome-face.png")
-
-export default function WelcomeScreen() {
-  const $bottomContainerInsets = useSafeAreaInsetsStyle(["bottom"])
-  const { theme, themed } = useAppTheme()
-
-  return (
-    <Screen safeAreaEdges={["top"]} contentContainerStyle={themed($container)}>
-      <View style={themed($topContainer)}>
-        <Image style={themed($welcomeLogo)} source={welcomeLogo} resizeMode="contain" />
-        <Text
-          testID="welcome-heading"
-          style={themed($welcomeHeading)}
-          tx="welcomeScreen:readyForLaunch"
-          preset="heading"
-        />
-        <Text tx="welcomeScreen:exciting" preset="subheading" />
-        <Image
-          style={$welcomeFace}
-          source={welcomeFace}
-          resizeMode="contain"
-          tintColor={theme.colors.palette.neutral900}
-        />
-      </View>
-
-      <View style={[themed($bottomContainer), $bottomContainerInsets]}>
-        <Text tx="welcomeScreen:postscript" size="md" />
-      </View>
-    </Screen>
-  )
+export default function Index() {
+  return <WelcomeScreen />
```

Comment:
Praise: I love this! Less duplicated code in the boilerplate to maintain! 🎉

Reply:
Woot think I can reduce it further, changes incoming 🙏